ios/Plugin/Plugin.swift

```diff
@@ -13,60 +13,67 @@ enum MyError: Error {
  */
 @objc(NativeAudio)
 public class NativeAudio: CAPPlugin {
-    
+
     var audioList: [String : Any] = [:]
     var fadeMusic = false
-    
+    var session = AVAudioSession.sharedInstance()
+
     public override func load() {
         super.load()
-        
+
         self.fadeMusic = false
-        
+
         do {
-            let session = AVAudioSession.sharedInstance()
-            try session.setCategory(AVAudioSession.Category.playback)
-            try session.setActive(false)
+            try self.session.setCategory(AVAudioSession.Category.ambient)
+            try self.session.setActive(false)
```

Comment:
Why we need change default mode from playback to ambient?

Reply:
What the difference of these mods?

Reply:
For playback:
`By default, using this category implies that your app’s audio is nonmixable—activating your session will interrupt any other audio sessions which are also nonmixable.`
https://developer.apple.com/documentation/avfaudio/avaudiosession/category/1616509-playback

For ambient:
`When you use this category, audio from other apps mixes with your audio.`
https://developer.apple.com/documentation/avfaudio/avaudiosession/category/1616560-ambient